Gatsby v4 Support

Hi @akabekobeko ! 👋 Hope you are well.

If I try upgrading to Gatsby v4, it seems like it cannot find the plugin:

```
Error: Cannot find module '/home/runner/work/project/node_modules/gatsby-remark-copy-relative-linked-files/index'
Require stack:
- /home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/validate.js
- /home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/load.js
- /home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/index.js
- /home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-config-and-plugins.js
- /home/runner/work/project/node_modules/gatsby/dist/services/initialize.js
- /home/runner/work/project/node_modules/gatsby/dist/services/index.js
- /home/runner/work/project/node_modules/gatsby/dist/bootstrap/index.js
- /home/runner/work/project/node_modules/gatsby/dist/commands/build.js
- /home/runner/work/project/node_modules/gatsby-cli/lib/create-cli.js
- /home/runner/work/project/node_modules/gatsby-cli/lib/index.js
- /home/runner/work/project/node_modules/gatsby-cli/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (/home/runner/work/project/node_modules/v8-compile-cache/v8-compile-cache.js:164:23)
    at /home/runner/work/project/node_modules/gatsby/src/bootstrap/load-plugins/validate.ts:234:52
    at Object.validate (/home/runner/work/project/node_modules/joi/lib/types/any.js:47:28)
    at Object.internals.rules (/home/runner/work/project/node_modules/joi/lib/validator.js:394:33)
    at Object.exports.validate (/home/runner/work/project/node_modules/joi/lib/validator.js:350:22)
    at internals.Base.$_validate (/home/runner/work/project/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/runner/work/project/node_modules/joi/lib/types/array.js:273:45)
    at Object.internals.rules (/home/runner/work/project/node_modules/joi/lib/validator.js:394:33)
    at Object.exports.validate (/home/runner/work/project/node_modules/joi/lib/validator.js:350:22)
    at internals.Base.$_validate (/home/runner/work/project/node_modules/joi/lib/base.js:773:26)
    at Object.validate (/home/runner/work/project/node_modules/joi/lib/types/keys.js:108:45)
    at Object.exports.validate (/home/runner/work/project/node_modules/joi/lib/validator.js:325:26)
    at Object.internals.entry (/home/runner/work/project/node_modules/joi/lib/validator.js:142:28)
    at Object.exports.entryAsync (/home/runner/work/project/node_modules/joi/lib/validator.js:58:30)
    at internals.Base.validateAsync (/home/runner/work/project/node_modules/joi/lib/base.js:553:26) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/validate.js',
    '/home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/load.js',
    '/home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-plugins/index.js',
    '/home/runner/work/project/node_modules/gatsby/dist/bootstrap/load-config-and-plugins.js',
    '/home/runner/work/project/node_modules/gatsby/dist/services/initialize.js',
    '/home/runner/work/project/node_modules/gatsby/dist/services/index.js',
    '/home/runner/work/project/node_modules/gatsby/dist/bootstrap/index.js',
    '/home/runner/work/project/node_modules/gatsby/dist/commands/build.js',
    '/home/runner/work/project/node_modules/gatsby-cli/lib/create-cli.js',
    '/home/runner/work/project/node_modules/gatsby-cli/lib/index.js',
    '/home/runner/work/project/node_modules/gatsby-cli/cli.js'
  ]
}
```

Do you have plans to support Gatsby v4?

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,3 +1,3 @@
 import { plugin } from './plugin'
 
-export default plugin
+module.exports = plugin

